enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -68,7 +68,7 @@ def to_representation(self, instance):
         representation['due_dates'] = []
         representation['pacing'] = course_overview['pacing']
         representation['org_name'] = course_overview['display_org_with_default']
-        representation['saved_for_later'] = instance.saved_for_later
+        representation['is_revoked'] = instance.license.is_revoked if instance.license else False
```

Comment:
do we want to be removing `saved_for_later` from this serializer?

Reply:
Now that `saved_for_later` is returned as its own course status, there's not really a reason for it to be in the serializer since it'd also be coming back with `course_run_status: "saved_for_later"`, and it felt odd to have both `course_run_status` and `saved_for_later` saying the same thing 🤷‍♂️ 

It was necessary before since `saved_for_later` courses were previously returning with `course_run_status: "completed"`, so `saved_for_later` in the response was the only way to differentiate.

Reply:
gotcha